### ISSUE TITLE
broken microcode not detected

### ISSUE THREAD

**Issue by wcwxyz** [speed47/spectre-meltdown-checker#165]
On my box, the microcode has version 0xb000025:
```
# cat /proc/cpuinfo | grep microcode | head -1
microcode       : 0xb000025
```
But the script is doing string comparing with predefined `0x0b000025`, which gives me:

```
  * CPU microcode is known to cause stability problems: (debug) is_ucode_blacklisted: no (79/1/0xb000025)
NO (model 79 stepping 1 ucode 0xb000025)
```

**Comment by wcwxyz** [comment]
I looked up linux kernel code regarding broken microcode version. It uses `microcode_version <= 0xb000025` to indicate broken microcode. You might want to change to that to align with upstream.

**Comment by speed47** [comment]
Thanks for the report.
Can you try the `ucode_decimal` branch and confirm that detection works correctly? I'm now doing a proper arithmetic comparison to avoid any other problems with string comparison of ucode versions.

**Comment by wcwxyz** [comment]
`ucode_decimal` branch works for me. Thanks for fixing it.
